package.json

```diff
@@ -33,6 +33,7 @@
   },
   "wireit": {
     "bidi-types": {
+      "clean": "if-file-deleted",
```

Comment:
I think you need to specify `files` for this to have any affect, which we can't do due to cddl living in different places for each use. Also I think we want to delete the files when we run this command (default behaviour).